### ISSUE TITLE
ld: unknown option: --unresolved-symbols=ignore-in-object-files

### ISSUE THREAD

**Issue by mustafaakin** [psanford/sqlite3vfs#6]
When I try to build a simple project in aarch64 in darwin I got the following error:

```sh
# github.com/psanford/sqlite3vfs
ld: unknown option: --unresolved-symbols=ignore-in-object-files
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any advice?

**Comment by psanford** [comment]
Thanks for reporting this! I pushed a change that should fix this in #7. Can you try it and let me know if it resolves the issue for you?

**Comment by psanford** [comment]
This is fixed by #7
